packages/flutter/lib/src/widgets/system_context_menu.dart

```diff
@@ -119,6 +159,258 @@ class _SystemContextMenuState extends State<SystemContextMenu> {
   @override
   Widget build(BuildContext context) {
     assert(SystemContextMenu.isSupported(context));
+
+    if (widget.items.isNotEmpty) {
+      final WidgetsLocalizations localizations = WidgetsLocalizations.of(context);
+      final List<IOSSystemContextMenuItemData> itemDatas =
+          widget.items
+              .map((IOSSystemContextMenuItem item) => item._getData(localizations))
+              .toList();
+      _systemContextMenuController.show(widget.anchor, itemDatas);
+    }
+
     return const SizedBox.shrink();
   }
 }
+
+/// Describes a context menu button that will be rendered in the system context
+/// menu and not by Flutter itself.
+///
+/// See also:
+///
+///  * [SystemContextMenu], a widget that can be used to display the system
+///    context menu.
+///  * [IOSSystemContextMenuItem], which performs a similar role but at the
+///    method channel level and mirrors the requirements of the method channel
+///    API.
+///  * [ContextMenuButtonItem], which performs a similar role for Flutter-drawn
+///    context menus.
+@immutable
+sealed class IOSSystemContextMenuItem {
+  const IOSSystemContextMenuItem();
+
+  /// The text to display to the user.
+  ///
+  /// Not exposed for some built-in menu items whose title is always set by the
+  /// platform.
+  String? get title => null;
+
+  /// Returns the representation of this class used by method channels.
+  IOSSystemContextMenuItemData _getData(WidgetsLocalizations? localizations);
+
+  @override
+  int get hashCode => title.hashCode;
+
+  @override
+  bool operator ==(Object other) {
+    if (identical(this, other)) {
+      return true;
+    }
+    if (other.runtimeType != runtimeType) {
+      return false;
+    }
+    return other is IOSSystemContextMenuItem && other.title == title;
+  }
+}
+
+/// Creates an instance of [IOSSystemContextMenuItem] for the system's built-in
+/// copy button.
+///
+/// Should only appear when there is a selection that can be copied.
+///
+/// The title and action are both handled by the platform.
+///
+/// See also:
+///
+///  * [SystemContextMenu], a widget that can be used to display the system
+///    context menu.
+///  * [IOSSystemContextMenuItemCopy], which specifies the data to be sent to
```

Comment:
I think this should reference `IOSSystemContextMenuItemDataCopy`. The rest of the `*Item` subclasses should also mention their `*Data` counterparts instead.